tests/periph_uart/Makefile

```diff
@@ -4,6 +4,7 @@ BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-uno nucleo-f031k6
 
 FEATURES_REQUIRED = periph_uart
 FEATURES_OPTIONAL = periph_lpuart  # STM32 L0 and L4 provides lpuart support
```

Comment:
not really part of this PR, but we might want to fix the above lines to use `+=` while we are add it...

Reply:
Fixed